tests/test_event_loops.py

```diff
@@ -160,7 +160,10 @@ def test_error(self):
     pass
 else:
 
-    @unittest.skipIf(IS_WINDOWS, "Windows is temporary not supported by TornadoEventLoop.")
+    @unittest.skipIf(
+        IS_WINDOWS,
+        "Windows is temporary not supported by TornadoEventLoop due to race conditions.",
```

Comment:
stable enough, maybe reason to enable some atomic tests before issue will be fully fixed